install/disconnected_install.adoc

```diff
@@ -277,6 +365,29 @@ $ docker pull registry.redhat.io/rhgs3/rhgs-volmanager-rhel7
 $ docker pull registry.redhat.io/rhgs3/rhgs-gluster-block-prov-rhel7
 $ docker pull registry.redhat.io/rhgs3/rhgs-s3-server-rhel7
 ----
+
+** For on-premise installations on IBM POWER8 or IBM POWER9 servers, run the following commands:
++
+[source, bash]
+----
+$ docker pull registry.access.redhat.com/openshift3/logging-auth-proxy:<tag>
```

Comment:
I'm pretty sure that these are the wrong packages.

Reply:
I broke this out to be both x86 and ppc64le, only x86 and only ppc64le (spoiler.. there aren't any ppc64le containers)

Also, I thought in 3.11 everything was in registry.redhat.io instead of registry.access.redhat.com?
```
#these are both ppc64le and x86
docker pull registry.redhat.io/openshift3/apb-base:<tag>
docker pull registry.redhat.io/openshift3/apb-tools:<tag>
docker pull registry.redhat.io/openshift3/automation-broker-apb:<tag>
docker pull registry.redhat.io/openshift3/csi-attacher:<tag>
docker pull registry.redhat.io/openshift3/csi-driver-registrar:<tag>
docker pull registry.redhat.io/openshift3/csi-livenessprobe:<tag>
docker pull registry.redhat.io/openshift3/csi-provisioner:<tag>
docker pull registry.redhat.io/openshift3/grafana:<tag>
docker pull registry.redhat.io/openshift3/image-inspector:<tag>
docker pull registry.redhat.io/openshift3/jenkins-2-rhel7:<tag>
docker pull registry.redhat.io/openshift3/jenkins-agent-maven-35-rhel7:<tag>
docker pull registry.redhat.io/openshift3/jenkins-agent-nodejs-8-rhel7:<tag>
docker pull registry.redhat.io/openshift3/jenkins-slave-base-rhel7:<tag>
docker pull registry.redhat.io/openshift3/local-storage-provisioner:<tag>
docker pull registry.redhat.io/openshift3/logging-fluentd:<tag>
docker pull registry.redhat.io/openshift3/manila-provisioner:<tag>
docker pull registry.redhat.io/openshift3/mariadb-apb:<tag>
docker pull registry.redhat.io/openshift3/mediawiki-apb:<tag>
docker pull registry.redhat.io/openshift3/mediawiki:<tag>
docker pull registry.redhat.io/openshift3/metrics-cassandra:<tag>
docker pull registry.redhat.io/openshift3/metrics-hawkular-openshift-agent:<tag>
docker pull registry.redhat.io/openshift3/metrics-heapster:<tag>
docker pull registry.redhat.io/openshift3/metrics-schema-installer:<tag>
docker pull registry.redhat.io/openshift3/mysql-apb:<tag>
docker pull registry.redhat.io/openshift3/node:<tag>
docker pull registry.redhat.io/openshift3/oauth-proxy:<tag>
docker pull registry.redhat.io/openshift3/ose-ansible-service-broker:<tag>
docker pull registry.redhat.io/openshift3/ose-ansible:<tag>
docker pull registry.redhat.io/openshift3/ose-cli:<tag>
docker pull registry.redhat.io/openshift3/ose-cluster-autoscaler:<tag>
docker pull registry.redhat.io/openshift3/ose-cluster-capacity:<tag>
docker pull registry.redhat.io/openshift3/ose-cluster-monitoring-operator:<tag>
docker pull registry.redhat.io/openshift3/ose-configmap-reloader:<tag>
docker pull registry.redhat.io/openshift3/ose-console:<tag>
docker pull registry.redhat.io/openshift3/ose-deployer:<tag>
docker pull registry.redhat.io/openshift3/ose-descheduler:<tag>
docker pull registry.redhat.io/openshift3/ose-docker-builder:<tag>
docker pull registry.redhat.io/openshift3/ose-docker-registry:<tag>
docker pull registry.redhat.io/openshift3/ose-egress-dns-proxy:<tag>
docker pull registry.redhat.io/openshift3/ose-egress-http-proxy:<tag>
docker pull registry.redhat.io/openshift3/ose-egress-router:<tag>
docker pull registry.redhat.io/openshift3/ose-haproxy-router:<tag>
docker pull registry.redhat.io/openshift3/ose-hyperkube:<tag>
docker pull registry.redhat.io/openshift3/ose-hypershift:<tag>
docker pull registry.redhat.io/openshift3/ose-keepalived-ipfailover:<tag>
docker pull registry.redhat.io/openshift3/ose-kube-rbac-proxy:<tag>
docker pull registry.redhat.io/openshift3/ose-kube-state-metrics:<tag>
docker pull registry.redhat.io/openshift3/ose-logging-curator5:<tag>
docker pull registry.redhat.io/openshift3/ose-logging-elasticsearch5:<tag>
docker pull registry.redhat.io/openshift3/ose-logging-eventrouter:<tag>
docker pull registry.redhat.io/openshift3/ose-logging-kibana5:<tag>
docker pull registry.redhat.io/openshift3/ose-metrics-server:<tag>
docker pull registry.redhat.io/openshift3/ose-node-problem-detector:<tag>
docker pull registry.redhat.io/openshift3/ose-operator-lifecycle-manager:<tag>
docker pull registry.redhat.io/openshift3/ose-ovn-kubernetes:<tag>
docker pull registry.redhat.io/openshift3/ose-pod:<tag>
docker pull registry.redhat.io/openshift3/ose-prometheus-config-reloader:<tag>
docker pull registry.redhat.io/openshift3/ose-prometheus-operator:<tag>
docker pull registry.redhat.io/openshift3/ose-recycler:<tag>
docker pull registry.redhat.io/openshift3/ose-service-catalog:<tag>
docker pull registry.redhat.io/openshift3/ose-template-service-broker:<tag>
docker pull registry.redhat.io/openshift3/ose-tests:<tag>
docker pull registry.redhat.io/openshift3/ose-web-console:<tag>
docker pull registry.redhat.io/openshift3/ose:<tag>
docker pull registry.redhat.io/openshift3/postgresql-apb:<tag>
docker pull registry.redhat.io/openshift3/prometheus-alert-buffer:<tag>
docker pull registry.redhat.io/openshift3/prometheus-alertmanager:<tag>
docker pull registry.redhat.io/openshift3/prometheus-node-exporter:<tag>
docker pull registry.redhat.io/openshift3/prometheus:<tag>
docker pull registry.redhat.io/openshift3/registry-console:<tag>
docker pull registry.redhat.io/openshift3/snapshot-controller:<tag>
docker pull registry.redhat.io/openshift3/snapshot-provisioner:<tag>
#containers only built for x86
docker pull registry.redhat.io/openshift3/ose-efs-provisioner:<tag>
docker pull registry.redhat.io/openshift3/metrics-hawkular-metrics:<tag>
docker pull registry.redhat.io/openshift3/jenkins-slave-maven-rhel7:<tag>
docker pull registry.redhat.io/openshift3/jenkins-slave-nodejs-rhel7:<tag>
#there aren't any pppc64le containers
```

Reply:
@jeffdyoung, your list is a little different than mine. Will you PTAL and let me know how you think we should proceed?

These images are in my list but not yours:
registry.redhat.io/openshift3/ose-control-plane:<tag>
registry.redhat.io/rhel7/etcd:3.2.22
several for cloudforms
several for rhgs3

These images are in your list but not mine:
registry.redhat.io/openshift3/ose:<tag>
registry.redhat.io/openshift3/ose-ovn-kubernetes:<tag>
registry.redhat.io/openshift3/manila-provisioner:<tag>
registry.redhat.io/openshift3/local-storage-provisioner:<tag>
registry.redhat.io/openshift3/ose-tests:<tag>

Reply:
@kalexand-rh Yeap, I messed these up, thanks for catching.  :)
> These images are in my list but not yours:

These containers are for both x86 and ppc64le
> registry.redhat.io/openshift3/ose-control-plane:
> registry.redhat.io/rhel7/etcd:3.2.22

cloudforms and rhgs3 are x86 only 
> several for cloudforms
> several for rhgs3
> 
> These images are in your list but not mine:

was supposed to be ose-control-plane.. so ignore this one 
> registry.redhat.io/openshift3/ose: 

These containers are for both x86 and ppc64le
> registry.redhat.io/openshift3/ose-ovn-kubernetes:
> registry.redhat.io/openshift3/manila-provisioner:
> registry.redhat.io/openshift3/local-storage-provisioner:
> registry.redhat.io/openshift3/ose-tests:

Reply:
Thanks @jeffdyoung! I've updated the package lists. Please let me know if you have more changes!